test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -201,6 +225,63 @@ public void testSignOutWhenSignedOut() {
         assertNull(cacheProvider.getUserSessionByUserId(userId));
     }
     
+    @Test
+    public void signUpWillCreateDataGroups() {
+        authService = spy(authService);
+        optionsService = spy(optionsService);
+        authService.setOptionsService(optionsService);
+        
+        Study tempStudy = TestUtils.getValidStudy(AuthenticationServiceImplTest.class);
+        tempStudy.setDataGroups(Sets.newHashSet("group1","group2","group3"));
+        tempStudy = studyService.createStudy(tempStudy);
```

Comment:
Yes because it's loading the spring context. This is one of those service classes that I will convert to mocks some day, but it already exists in this form.